### PR TITLE
[SW-851] Update design of Account selection modal

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
sw-851

File: src/popup/components/FormSelectOptionsHeader.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="form-select-options-header">
+    <span class="text-heading-3 title">{{ title }}</span>
+    <InputSearch
+      v-if="options.length > 3"
```
Comment (konradodo) [inline src/popup/components/FormSelectOptionsHeader.vue:5]: 3 -> Move that values to constans

**Review by konradodo — CHANGES_REQUESTED** [review]

File: src/popup/components/FormSelectOptionsHeader.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      if (searchTerm.value) {
+        const searchTermLower = searchTerm.value.toLowerCase();
+        const result = props.options.filter(({ text, value }) => (
+          text.toLowerCase().includes(searchTermLower)
+          || value.toString().includes(searchTermLower)
+        ));
+
+        emit('search', result);
+      }
+      emit('search', props.options);
```
[the remark this answers is not in the document]
Comment (pstachel) [reply src/popup/components/FormSelectOptionsHeader.vue:40]: Good catch, missed that one

**Review by konradodo — APPROVED** [review]

**Review by konradodo — APPROVED** [review]

File: src/popup/components/FormSelectOptionsHeader.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  components: { InputSearch },
+  props: {
+    value: { type: String, default: '' },
+    title: { type: String, default: '' },
+    searchPlaceholder: { type: String as PropType<String | TranslateResult>, default: '' },
+    options: { type: Array as PropType<IFormSelectOption[]>, default: () => [] },
+  },
+  setup(props, { emit }) {
+    const searchTerm = ref('');
+    function filterOptions(searchPhrase: string) {
```
Comment (peronczyk) [inline src/popup/components/FormSelectOptionsHeader.vue:32]: This component should not do the actual filtering. Instead it should emit the search phrase and the parent component should react on this.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/TokenAmount.vue [hunk 4]
```diff
@@ -141,5 +144,13 @@ export default defineComponent({
       @extend %face-sans-18-regular;
     }
   }
+
+  &.align-right {
+    text-align: right;
+
+    .fiat {
+      @extend %face-sans-12-regular;
+    }
```
Comment (peronczyk) [inline src/popup/components/TokenAmount.vue:153]: I think it's bad to change the font weight of elements if we are changing the alignment. Those 2 things are not connected.

File: src/popup/components/TokenAmount.vue [hunk 5]
```diff
@@ -57,6 +57,7 @@ export default defineComponent({
     noSymbol: Boolean,
     highPrecision: Boolean,
     dynamicSizing: Boolean,
+    alignRight: Boolean,
```
Comment (peronczyk) [inline src/popup/components/TokenAmount.vue:60]: Why not using the CSS text-align on the parent level as this component is not a block but inline element?

**Review by peronczyk — APPROVED** [review]

**Pull request merged**